.prebuildrc

```diff
@@ -1,7 +1,7 @@
 {
   "target": [
-    "22.15.1",
-    "24.0.2"
+    "22.17.0",
```

Comment:
You can leave these changes in the diff but I'd like to know if this was tweaked on a hunch or did you observe a difference?

Reply:
I just used the latest of everything, there's no difference. Same with the upgrade of the github action and of the prebuild dependency - I didn't want to bother with old versions.

The header file was added because of consistency and also because I was hoping to get v24 to work, but it seems the linker has an issue (does not like the signature of SourceLocation) and I didn't have an idea how to fix it.